libs/platform/src/lib/components/info-label/info-label.component.html

```diff
@@ -1,3 +1,10 @@
-<span fd-info-label [type]="type" [color]="color" [glyph]="glyph" [attr.aria-label]="ariaLabel" [attr.aria-labelledby]="ariaLabelledBy">
-	<ng-content></ng-content>
-</span>
\ No newline at end of file
+<fd-info-label
+    [type]="type"
+    [color]="color"
+    [glyph]="glyph"
+    [attr.aria-label]="ariaLabel"
+    [attr.aria-labelledby]="ariaLabelledBy"
+    [title]="title"
```

Comment:
Why is in this case `title` and in the other cases `attr.title`?

Reply:
`attr.title` is already added in the core component in order not to duplicate i have added like as` title`